extras/scaler/internal/gcp/startup.sh

```
@@ -166,60 +166,81 @@ if [ "$current_runner_version" != "$RUNNER_VERSION" ]; then
+  # with "Failed to initialize NVML: Unknown Error".
+  # See: https://github.com/NVIDIA/nvidia-docker/issues/1730
+  log "  Creating /dev/char symlinks..."
+  if ctk_out="$(nvidia-ctk system create-dev-char-symlinks --create-all 2>&1)"; then
+    log "  /dev/char symlinks created."
+  else
+    log "WARNING: Failed to create /dev/char symlinks: ${ctk_out}"
+  fi
+
+  # Verify GPU devices
+  log "  GPU devices:"
+  ls -la /dev/nvidia* 2>&1 | while read -r line; do log "    $line"; done || true
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)